js/editor-cm.js

```diff
@@ -8,20 +111,30 @@ var EditSession = CodeMirror.Doc;
 function EditorCodeMirror(editorElement, settings) {
   this.element_ = editorElement;
   this.settings_ = settings;
+
+
   this.cm_ = CodeMirror(
       editorElement,
       {
         'value': '',
         'autofocus': true,
         'matchBrackets': true,
+        'inputStyle': 'devToolsAccessibleTextArea',
+        // Poll interval of ~25 days, this is to prevent codemirror from
+        // deleting all of our selections
+        'pollInterval': Math.pow(2, 31) - 1,
```

Comment:
does this break anything like IME input? also add to comment the units pollInterval is taking e.g., '25 days (in milliseconds).

Reply:
Code was deleted from this PR